tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -232,3 +232,136 @@ def get_resource_params(self, m, r):
                                               r['selfLink']).groups()
         return {'project': project,
                 'instanceTemplate': instance_template}
+
+
+@resources.register('autoscaler')
+class Autoscaler(QueryResourceManager):
+    """GCP resource: https://cloud.google.com/compute/docs/reference/rest/v1/autoscalers"""
+    class resource_type(TypeInfo):
+        service = 'compute'
+        version = 'v1'
+        component = 'autoscalers'
+        id = 'name'
+        enum_spec = ('aggregatedList', 'items.*.autoscalers[]', None)
+
+        @staticmethod
+        def get(client, resource_info):
+            project, zone, autoscaler = re.match(
+                'projects/(.*?)/zones/(.*?)/autoscalers/(.*)',
+                resource_info['resourceName']).groups()
+
+            return client.execute_command(
+                'get', {'project': project,
+                        'zone': zone,
+                        'autoscaler': autoscaler})
+
+
+@Autoscaler.action_registry.register('set')
+class AutoscalerSet(MethodAction):
+    """
+    `Patches <https://cloud.google.com/compute/docs/reference/rest/v1/autoscalers/patch>`_
+    configuration parameters for the autoscaling algorithm.
+
+    The `coolDownPeriodSec` specifies the number of seconds that the autoscaler
+    should wait before it starts collecting information from a new instance.
```

Comment:
please don't duplicate docs for api method parameters, just leave the link to the method.

Reply:
compared to the docs info for these attributes, and the docs inline here seem quite a bit simpler than those, so lets leave as is.